<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(87, 106, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTreeWidget" name="friends_treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>521</height>
    </rect>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">好友列表</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QTextEdit" name="show_msg_textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>531</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>20</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat_textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>481</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_Btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="myname_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="friendname_label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="audio_Btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>语音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_up_Btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msg_log_Btn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>记录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
